src/core/Akka.Persistence/Akka.Persistence.csproj

```diff
@@ -112,4 +114,118 @@
   </Target>
   -->
   
+  <Choose>
```

Comment:
Bah! I missed this when I first looked at the PR and merged it.

We'll need to remove that `<Choose>` block below.